### DISCUSSION TITLE
유니티 월드 초대시 네트워크 문제

### DISCUSSION THREAD

**Discussion by strangestraw** [naverz/zepeto-studio-kor#356] [5. Q&A]
안녕하세요. 유니티로 생성한 월드에서 초대 시에 에러가 발생하는 경우가 있습니다.
또는 링크 복사/공유를 통한 입장 시에 에러가 발생하는 경우가 있는데요.
이 경우, 제페토 서버의 문제인지 혹은 다른 처리가 필요한 것인지 궁금합니다.
제페토 서버의 status를 확인할 수 있는 방법이 있으면 알려주세요.

![image](https://user-images.githubusercontent.com/5335151/153334232-d50a93d9-8455-45b3-9e46-b878f9ee2931.png)
![image](https://user-images.githubusercontent.com/5335151/153334248-4703f37b-4925-4d67-97aa-82d8164237d4.png)

**Comment by hyeonjin-jo** [comment]
안녕하세요.

현재 내부에서 파악하고 있는 상황으로 가능한 빠른 시일안에 수정 진행하겠습니다.

문의주셔서 감사합니다

**Reply by strangestraw** [reply]
확인 감사합니다. 수정되거나 진척사항 있으시면 공유 부탁드립니다.
